# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hplip/sanify.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- hplip-2.8.2/scan.py.vanilla	2008-02-19 15:33:13.000000000 +0100
+++ hplip-2.8.2/scan.py	2008-02-19 15:33:32.000000000 +0100
@@ -998,7 +998,7 @@
     device.setOption("resolution", res)
     
     if adf:
-        device.setOption("source", "ADF")
+        device.setOption("source", "ADF Front")
         device.setOption("batch-scan", True)
     else:
         try:
--- hplip-2.8.2/scan/sane/common.h.vanilla	2008-02-19 15:32:47.000000000 +0100
+++ hplip-2.8.2/scan/sane/common.h	2008-02-19 15:33:05.000000000 +0100
@@ -48,7 +48,7 @@
 
 #define STR_ADF_MODE_AUTO SANE_I18N("Auto")
 #define STR_ADF_MODE_FLATBED  SANE_I18N("Flatbed")
-#define STR_ADF_MODE_ADF  SANE_I18N("ADF")
+#define STR_ADF_MODE_ADF  SANE_I18N("ADF Front")
 
 #define STR_TITLE_ADVANCED SANE_I18N("Advanced")
 
